python/ray/tests/test_basic.py

```diff
@@ -1114,6 +1114,12 @@ def test_import_ray_does_not_import_grpc():
     import grpc  # noqa: F401
 
 
+# https://github.com/ray-project/ray/issues/36431
+def test_temp_dir_must_be_absolute(shutdown_only):
+    # This test fails with the flag set to false.
```

Comment:
which flag you are referring to?

Reply:
it was copy pasted from the other test. Updated the comment.